crates/ruff_python_formatter/tests/snapshots/format@statement__delete.py.snap

```diff
@@ -146,6 +151,8 @@ del (
     a,
 )  # Trailing
 
+del a, a
```

Comment:
Huh. Actually `black` will format this as 
```py
del (x, x)
```

Reply:
Maybe this is because it could be interpreted as `del tuple(x, x)` instead of `del obj, obj`?

A more reasonable example would be
```py
a, b, c = 1, 2, 3
del (a, b, c)
```
But I'd guess that's not actually performing a *delete* of `a`, `b`, and `c` but instead a new object of `tuple(a, b, c)`

Reply:
Makes sense
```
>>> print(ast.dump(ast.parse('del (a, b, c)'), indent=4))
Module(
    body=[
        Delete(
            targets=[
                Tuple(
                    elts=[
                        Name(id='a', ctx=Del()),
                        Name(id='b', ctx=Del()),
                        Name(id='c', ctx=Del())],
                    ctx=Del())])],
    type_ignores=[])
```

Reply:
695df60cf4d1d4d2067749322d669b2c99f757ed